cmd/chisel/cmd_cut.go

```diff
@@ -82,6 +85,43 @@ func (cmd *cmdCut) Execute(args []string) error {
 		return err
 	}
 
+	archives, err := cmd.packageArchives(release)
+	if err != nil {
+		return err
+	}
+
+	report, err := slicer.Run(&slicer.RunOptions{
+		Selection: selection,
+		Archives:  archives,
+		TargetDir: cmd.RootDir,
+	})
+	if err != nil {
+		return err
+	}
+
+	manifestDir, err := findManifestDir(selection.Slices)
+	if err == nil {
```

Comment:
Just saw this one, it is not a good pattern to use an error to see if a path is there. Normally you would check `err != nil` and return error, then check `manifestDir != ""` or include a boolean flag in the return values.

After the refactor this will probably be a list so the loop will automatically work for 0 items but, code-wise, the advice is more general.

Reply:
Good point. This has since been changed.